php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1104,22 +1104,6 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     abstract function getSubtestList(): array;
 
-    /**
-     * Marks an element in the lorisform object as being required (for
-     * use with the requiredIf rule)
-     *
-     * @param string $elementName the element to add to the required list
-     *
-     * @return void
-     */
-    function setRequired(string $elementName): void
-    {
-        if (!in_array($elementName, $this->form->_required)) {
```

Comment:
LorisForm doesn't declare a _required, and makes no references to one. If this function is ever called by anyone it wouldn't do anything.